crates/next-core/src/next_client/context.rs

```diff
@@ -397,21 +397,43 @@ pub async fn get_client_module_options_context(
     Ok(module_options_context)
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Hash, TaskInput, TraceRawVcs, Serialize, Deserialize)]
+pub struct ClientChunkingContextOptions {
+    pub mode: Vc<NextMode>,
```

Comment:
why are we storing `Vc` instead of `ResolvedVc` in this struct.... that seems surprising (i thought it was disallowed?)